components/Step.vue

```diff
@@ -55,6 +59,13 @@
 </template>
 
 <script setup>
+  function truncate(text, maxLength) {
```

Comment:
remember this function, I guess you will use the same in the TreeView? If so, you should move it somewhere. @JulienChampagnol where is the bast location? utils ?

Reply:
Yeah would say utils